docs/lua-records/reference/comboaddress.rst

```diff
@@ -19,6 +19,13 @@ Functions and methods of a ``ComboAddress``
 
     addr = newCA("1.2.3.4")
 
+.. function:: newCAFromRaw(rawaddress[, port]) -> ComboAddress
+
+  Returns a new :class:`ComboAddress` object based on the 4- or 16-octet string.
+
+  :param string rawaddress: The IPv4 of IPv6 address as a 4/16 octet string
```

Comment:
Example or more explanation needed, it's not clear how to encode the string

Reply:
added an example